src/FluxCalculator.jl

```diff
@@ -492,51 +479,70 @@ function compute_surface_fluxes!(
     inputs = FluxCalculator.surface_inputs(surface_scheme, input_args)
 
     # calculate the surface fluxes
-    fluxes = FluxCalculator.get_surface_fluxes!(inputs, surface_params, area_fraction)
-    (; F_turb_ρτxz, F_turb_ρτyz, F_shf, F_lhf, F_turb_moisture) = fluxes
+    fluxes = FluxCalculator.get_surface_fluxes!(inputs, surface_params)
+    (; F_turb_ρτxz, F_turb_ρτyz, F_shf, F_lhf, F_turb_moisture, L_MO, ustar, buoyancy_flux) = fluxes
+
+
+    # Zero out fluxes where the area fraction is zero
+    # Multiplying by `area_fraction` is not sufficient because the area fraction
+    # may be approximately, but not exactly, zero.
```

Comment:
Maybe we should just set `area_fraction` to 0 when it is close to zero.

Reply:
Something else must be going on because when I do that and just multiply fluxes by `area_fraction` instead of explicitly making these checks, the atmosphere is stable when it first steps

Reply:
I think that this is worth understanding better

Reply:
I looked into it more. Multiplying by 0 doesn't work because we sometimes have NaNs in the fluxes where the area fraction is 0. We will need to manually substitute 0 anywhere the area fraction is 0, and if/when we want to support fractional area fractions, we'll need to also multiply by the area fraction

Reply:
Okay, thanks for looking into it!